app/components/templates/templates.html

```diff
@@ -55,15 +55,18 @@
           <!-- !network-input -->
           <!-- env -->
           <div ng-repeat="var in state.selectedTemplate.Env" ng-if="!var.set" class="form-group">
-            <label for="field_{{ $index }}" class="col-sm-2 control-label text-left">{{ var.label }}</label>
+            <label for="field_{{ $index }}" class="col-sm-2 control-label text-left">{{ var.label }}<portainer-tooltip ng-if="var.hint" position="bottom" message="{{ var.hint }}"></portainer-tooltip></label>
             <div class="col-sm-10">
               <select ng-if="applicationState.endpoint.mode.provider !== 'DOCKER_SWARM' && var.type === 'container'" ng-options="container|containername for container in runningContainers" class="form-control" ng-model="var.value">
                 <option selected disabled hidden value="">Select a container</option>
               </select>
               <select ng-if="applicationState.endpoint.mode.provider === 'DOCKER_SWARM' && var.type === 'container'" ng-options="container|swarmcontainername for container in runningContainers" class="form-control" ng-model="var.value">
                 <option selected disabled hidden value="">Select a container</option>
               </select>
-              <input ng-if="!var.type || !var.type === 'container'" type="text" class="form-control" ng-model="var.value" id="field_{{ $index }}">
+              <input ng-if="!var.values && (!var.type || !var.type === 'container')" type="text" class="form-control" ng-model="var.value" id="field_{{ $index }}">
+              <select ng-if="var.values" class="form-control" ng-model="var.value" id="field_{{ $index }}">
+                  <option ng-repeat="(value, name) in var.values" value="{{ value }}">{{ name }}</option>
```

Comment:
We could add a placeholder to guide user like this one:
```html
<option selected disabled hidden value="">Select value</option>
```

Reply:
Nice idea.